Cargo.toml

```diff
@@ -76,3 +76,7 @@ version = "610.0.0"
 package = "rustc-ap-syntax_pos"
 version = "610.0.0"
 
+[dependencies.rustc_data_structures]
+package = "rustc-ap-rustc_data_structures"
```

Comment:
This was already a part of the dependency tree (since nearly every `rustc-ap-*` package uses it), but adding it directly to gain access to one of the traits for use in the parser emitter.